client/interfaces.ts

```diff
@@ -646,6 +646,7 @@ export interface IPlanningCoverageItem {
     version_creator: string;
     firstcreated: string;
     versioncreated: string;
+    add_coverage_to_workflow: boolean;
```

Comment:
Can't it be checked whether coverage is in workflow using other fields? If so, maybe we don't need an extra one that stores computed data?